baybe/parameters/base.py

```diff
@@ -16,6 +17,7 @@
     get_base_structure_hook,
     unstructure_base,
 )
+from baybe.utils.basic import classproperty
```

Comment:
using this might cause doc problems again which might not easily be resolved because this is not private this time...

Reply:
Rgiht, could be. Let's rebase on #282 once merged and see how far the pipeline runs. In the worst case, I make it a breaking change. Or do you have another clever idea?

Reply:
yes if problem make it breaking imo fine

Reply:
indeed not working =/ let me turn it into breaking change and ping you again once done

Reply:
@Scienfitz @AVHopp: now change, please have another look